src/pykx/pandas_api/pandas_meta.py

```diff
@@ -134,6 +134,11 @@ def shape(self):
     @property
     def size(self):
         return q('{count[x] * count[cols x]}', self)
+    
+    @property
+    def values(self):
+        tab = self
+        return q('value peach', tab)
```

Comment:
Not completely sure `peach` does anything here. I would have to ask the devs if multithreading inside the q session is supported in PyKX, since I have seen little use of `peach` on the library and by default q runs singlethreaded (see https://code.kx.com/q/wp/multi-thread/#setting-up-kdb-for-parallel-processing). If it turns out it's not really used, then (minor) consider either of these implementations:

```python
kx.q("(value')", q_table)
```
Or its pythonic counterpart:

```python
kx.q.each(kx.q.value, q_table)
```
If it turns out that multithreading inside the q session is supported, then (minor) consider the pythonic implementation:

```python
kx.q.peach(kx.q.value, q_table)
```

Either way, I will try to ask this question to the devs next time we talk to them and answer this comment with whatever they tell me so you can decide what to do next.

Reply:
Just asked the PyKX team about this. Turns out that the library itself handles the multithreading. Generally speaking PyKX is greedy, so it takes everything it can in terms of resources, but when dealing with q sessions we should focus on using `each`. If I understood correctly, in the future they will have a sort of flag that will change the behavior of `each` into `peach`.

TLDR: We should use `each`.

Feel free to use the solution you feel  fits best.